src/proxymatic/discovery/marathon.py

```diff
@@ -226,6 +235,8 @@ def failed(check):
                     # Set backend load balancer options
                     self._applyBackendAttributeInt('weight', taskConfig, portIndex, server)
                     self._applyBackendAttributeInt('maxconn', taskConfig, portIndex, server, self._groupsize)
+                    self._applyBackendConnectionTimeout('timeoutclient', taskConfig, portIndex, server)
```

Comment:
Remove these lines and then set the attribute on the service object (this code sets the attribute on the task/container object instead of the service)

Instead add at line ~248

```
                    # Apply timeout parameters
                    self._applyAttributeInt('timeout.client', taskConfig, portIndex, services[key])
                    self._applyAttributeInt('timeout.server', taskConfig, portIndex, services[key])
```